lepton/azure_network.go

```diff
@@ -285,12 +285,19 @@ func (a *Azure) CreateNetworkSecurityGroup(ctx context.Context, location string,
 
 	securityRules := make([]network.SecurityRule, len(c.RunConfig.Ports))
 
+	var communicationProtocol network.SecurityRuleProtocol
+	if c.RunConfig.UDP {
```

Comment:
i think the convention we are using right now is tcp && udp not tcp || udp - maybe this should change in the future but prob. best to keep the same for now

Reply:
prob. we should use flags to distinguish the ports which use tcp or udp (--tcp and --udp) instead of a configuration variable

Reply:
i'm in favor of adding a --udp flag, although tcp is 99% going to be the default so I think for now we should just keep it as an addition, so:

1) make this be tcp+udp if specified
2) add support for --udp for ease of use

Reply:
We're using `-p` in some commands to provide the configuration value for cloud provider project. I think `ops instance create` is the only one which uses `-p` to defined the instance port. I know it may be difficult to keep the consistency across all commands, but we can use this opportunity to change the port flag.